src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckCollector.java

```diff
@@ -236,4 +246,11 @@ protected LinkableItem getSeverity(String vulnerabilityUrl, boolean partOfKey) {
         return vulnerabilityViewMap;
     }
 
+    protected ComponentItemPriority getPolicyPriority(String severity) {
+        if (StringUtils.isBlank(severity) || !policyPriorityMap.containsKey(severity.trim().toLowerCase())) {
```

Comment:
We could check if it is not blank and then store the result of `.trim()` and `.toLowerCase()` before getting from the map. Then we could make `return ComponentItemPriority.NONE` the last line of the method outside of the if-statement.